projects/devices/robotiq/protos/Robotiq3fGripper.proto

```diff
@@ -1,11 +1,11 @@
-#VRML_SIM R2020a utf8
+#VRML_SIM R2020b utf8
 # license: Apache License 2.0
 # license url: http://www.apache.org/licenses/LICENSE-2.0
 # documentation url: https://www.cyberbotics.com/doc/guide/universal_robots
 # A 3-finger adaptive robot gripper from ROBOTIQ.
 
 PROTO Robotiq3fGripper [
-  field   SFVec3f    translation    0 0 0
+  field   SFVec3f    translation    0 0.04 0
```

Comment:
Why changing the default translation?

Reply:
To correctly attach the gripper. The base link origin is in the middle of the gripper:
![image](https://user-images.githubusercontent.com/52213164/86912947-28729900-c11e-11ea-9432-fbb6796d2e8a.png)